### PR TITLE
Update links to blog posts to newer blogs

### PR DESCRIPTION
**Pull request by edwardhorsford**
Updates to point 6 of the design principles to link to newer blog posts on accessibility.

### CODE REVIEW

File: app/views/root/designprinciples.html.erb [hunk 1]
```diff
@@ -180,9 +180,10 @@
             think about those people from the start.
            </p>
            <ul>
+             <li><a href="https://accessibility.blog.gov.uk/2016/05/16/what-we-mean-when-we-talk-about-accessibility-2/"></a>What we mean when we talk about accessibility, by&nbsp;Alistair&nbsp;Duggin</li>
+             <li><a href="https://accessibility.blog.gov.uk/2016/05/16/consider-the-range-of-people-that-will-use-your-product-or-service/">Consider the range of people that will use your product or service, by by&nbsp;Alistair&nbsp;Duggin</a></li>
```
Comment (gemmaleigh) [inline app/views/root/designprinciples.html.erb]: Looks like a copy and paste error - there's a double, "by by" Alistair. Once fixed I'm happy to merge.

**Comment by edwardhorsford** [comment]
@gemmaleigh fixed. Thanks!

File: app/views/root/designprinciples.html.erb [hunk 2]
```diff
@@ -180,9 +180,10 @@
             think about those people from the start.
            </p>
            <ul>
+             <li><a href="https://accessibility.blog.gov.uk/2016/05/16/what-we-mean-when-we-talk-about-accessibility-2/"></a>What we mean when we talk about accessibility, by&nbsp;Alistair&nbsp;Duggin</li>
```
Comment (gemmaleigh) [inline app/views/root/designprinciples.html.erb]: More nits, sorry! Looks like you'll need the closing </a> after the name of the article, but before the name of the author - to keep with the convention for the others.

**Comment by edwardhorsford** [comment]
@gemmaleigh fixed! I also put the last anchor in an `li` which it looks like I'd forgotten to do.

**Comment by gemmaleigh** [comment]
@edwardhorsford, thanks!

**Pull request merged**